graylog2-server/src/main/java/org/graylog/plugins/views/search/views/ViewDTO.java

```diff
@@ -48,7 +50,9 @@
 @AutoValue
 @JsonDeserialize(builder = ViewDTO.Builder.class)
 @WithBeanGetter
+@DbEntity(collection = "dashboards", readPermission = RestPermissions.DASHBOARDS_READ)
```

Comment:
Could we add a comment here explaining that the entity is based on a MongoDB view? Without context, I think it is confusing that were are mentioning a different collection here than two lines below.

Reply:
:white_check_mark: